lib/lndclient/LndClient.ts

```diff
@@ -208,8 +208,8 @@ class LndClient extends BaseClient {
    * Return general information concerning the lightning node including it’s identity pubkey, alias, the chains it
    * is connected to, and information concerning the number of open+pending channels.
    */
-  private getInfo = (): Promise<lndrpc.GetInfoResponse.AsObject> => {
-    return this.unaryCall<lndrpc.GetInfoRequest, lndrpc.GetInfoResponse.AsObject>('getInfo', new lndrpc.GetInfoRequest());
+  public getInfo = (): Promise<lndrpc.GetInfoResponse> => {
```

Comment:
@sangaman so you like me to revert the getInfo code to use object? I can do it. I made the change since I need now to call getInfo to get blockHeight when checking CLTV for the maker to taker route.
note that getInfo was a private method with no external usage.


Do you think I should also do the same for queryRoutes? transform the routes to objects and rebuild them for sendToRoutes? Big NO on this.

Reply:
No, I wouldn't say you have to change `queryRoutes` since it's new code anyway and a core piece of this PR. If you even just want to split this into a separate commit but keep it in the PR that'd be better, but refactoring everything in a single PR would be cleaner imo. I'm mainly just thinking about maintaining a sensible commit history which I think is becoming increasingly important.